dev-infrastructure/modules/mgmt-kv-access.bicep

```diff
@@ -1,5 +1,5 @@
-@description('Cluster user assigned identity resource id, used to grant KeyVault access')
-param clusterServiceMIResourceId string
+@description('Managed identity resource id that gets access to the specified KeyVaults.')
+param managedIdentityResourceId string
```

Comment:
Curious as to why we're renaming an existing managed identity rather than creating a unique one for the refresher. Is the goal to re-use this one across multiple components that require workload identity?

Reply:
In any case, don't we also need to create a new federated identity credential targeting the refresher's service account / namespace / etc...?